src/pages/cluster-management/cluster-configuration.html

```diff
@@ -0,0 +1,143 @@
+                                <input class="form-control" placeholder="This field is required" type="number" ng-model="clusterConfiguration.messageSizeKB"
+                                       required id="message-size">
+                            </div>
+                        </div>
+                        <div class="form-group row">
+                            <label for="verification-timeout" class="col-lg-6 col-xl-5 col-form-label">{{'cluster_management.cluster_configuration_properties.verification_timeout' | translate}}*</label>
+                            <div class="col-lg-6 col-xl-7" tooltip="{{'cluster_management.cluster_configuration_properties.verification_timeout_tooltip' | translate}}">
+                                <input class="form-control" placeholder="This field is required" type="number" ng-model="clusterConfiguration.verificationTimeout"
+                                       required id="verification-timeout">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-lg-6">
+                        <h5>{{'cluster_management.cluster_page.manual_node_add' | translate}}</h5>
+                        <div class="form-group row">
+                            <label for="node-rpc-address" class="col-lg-5 col-xl-5 col-form-label">{{'cluster_management.cluster_page.node_address' | translate}}</label>
+                            <div class="col-lg-7 col-xl-7" tooltip="{{'cluster_management.cluster_page.node_address_tooltip' | translate}}">
+                                <input class="form-control" placeholder="{{'cluster_management.cluster_page.node_address_placeholder' | translate}}"
+                                       type="text" ng-model="rpcAddress" id="node-rpc-address">
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <button type="button" class="btn btn-primary addNodeBtn"
+                                        ng-click="addNodeToList(rpcAddress)" ng-disabled="!rpcAddress"
+                                        tooltip="{{'cluster_management.cluster_page.add_node_btn_tooltip' | translate}}"
+                                        tooltip-placement="top">
+                                    <span class="icon-plus"></span>
+                                    {{'cluster_management.cluster_page.add_node_btn' | translate}}
+                                </button>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <div ng-if="clusterConfiguration.nodes.includes(rpcAddress)" class="nodeAlreadyAdded alert alert-danger mt-1">
+                                    {{'cluster_management.cluster_page.errors.node_in_list' | translate}}
```

Comment:
`nodeAlreadyAdded` why camelcase and not dash separated?

Reply:
I saw it like this and continued. I made them dash separated.